### ISSUE TITLE
Statistics

### ISSUE THREAD

**Issue by b3n4kh** [ait-cs-IaaS/koord2ool#2]
For some types of questions, statistics (like mean, std.dev, etc.) should be included for responses where the question implies a numerical response.

**Comment by otmarlendl** [comment]
Yes, for my testing I used the question "What's the temperature?" as a test. It doesn't make sense to list how often a certain value was given, the statistical properties make a lot more sense. 

A simple mean would be good, Box & Whisker charts even better.

But the math isn't so trivial:

* for a simple mean, you can do a running mean, where each new response triggers a new value for the average

* But box plots need a defines aggregation interval to which the box applies to. That's not trivial, e.g. what is that interval and should older answers carry on into the given interval?

So I recommend to do the running mean first.
